Scoop/Controller/ProfileViewController.swift

```diff
@@ -329,7 +343,7 @@ class ProfileViewController: UIViewController, ProfileViewDelegate {
         })
     }
     
-    func updateProfile() {
+    func updateUserProfile() {
         NetworkManager.shared.getUser { result in
```

Comment:
I see that you used `self`in the lines below. Create a weak reference to `self` in this networking closure and unwrap a strong reference.